travis/cfg/travis_run_pylint_exclude_100.cfg

```
@@ -1,5 +1,5 @@
 [MASTER]
-ignore=account_parent,web_gantt_native,web_gantt_project_native,html_tree_widget,web_hide_db_manager_link,web_widget_colorpicker,web_widget_time_delta,wk_wizard_messages,odoo_shipping_service_apps,ups_delivery_carrier,amco_ups_config_settings_mx,dev_partner_destination_location,l10n_mx_account_tree,account_period_and_fiscalyear,experts_account_invoice_cfdi_33_data,experts_stock_lot_invoice,experts_account_accountinge,experts_account_invoice_cfdi_33,experts_account_cfdi_cce,experts_account_voucher_cfdi_33,experts_account_diot,experts_account_cfdi_cce_data,experts_invoice_report,smile_redis_session_store,gts_stock_xlsx_report_valuation,
+ignore=account_parent,web_gantt_native,web_gantt_project_native,html_tree_widget,web_hide_db_manager_link,web_widget_colorpicker,web_widget_time_delta,wk_wizard_messages,odoo_shipping_service_apps,ups_delivery_carrier,amco_ups_config_settings_mx,dev_partner_destination_location,l10n_mx_account_tree,account_period_and_fiscalyear,experts_account_invoice_cfdi_33_data,experts_stock_lot_invoice,experts_account_accountinge,experts_account_invoice_cfdi_33,experts_account_cfdi_cce,experts_account_voucher_cfdi_33,experts_account_diot,experts_account_cfdi_cce_data,experts_invoice_report,smile_redis_session_store,gts_stock_xlsx_report_valuation,fedex_delivery_carrier,
```

This line has become too long, could it be reduced as so?
```suggestion
ignore= account_parent,
web_gantt_native,
web_gantt_project_native,
html_tree_widget,
web_hide_db_manager_link,
web_widget_colorpicker,
web_widget_time_delta,
wk_wizard_messages,
odoo_shipping_service_apps,
ups_delivery_carrier,
amco_ups_config_settings_mx,
dev_partner_destination_location,
l10n_mx_account_tree,
account_period_and_fiscalyear,
experts_account_invoice_cfdi_33_data,
experts_stock_lot_invoice,
experts_account_accountinge,
experts_account_invoice_cfdi_33,
experts_account_cfdi_cce,
experts_account_voucher_cfdi_33,
experts_account_diot,
experts_account_cfdi_cce_data,
experts_invoice_report,
smile_redis_session_store,
gts_stock_xlsx_report_valuation,
fedex_delivery_carrier,
```

---

I'm not sure it can, because this is not python, it's a .cfg file, I can try